staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -158,6 +159,9 @@ type Store struct {
 	// ExportStrategy implements resource-specific behavior during export,
 	// optional. Exported objects are not decorated.
 	ExportStrategy rest.RESTExportStrategy
+	// TableConvertor is an optional interface for transforming items or lists
+	// of items into tabular output. If unset, the default will be used.
+	TableConvertor rest.TableConvertor
```

Comment:
ExportStrategy and Tables are the same.  Store isn't about CRUD, it's about REST.  Export/Table are Representations of State.